OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -401,8 +401,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </collation>
                                 
                                 <condition key="good">The first leaf and the outer margins of many pages are stained by dirt, e.g. 
-                                <locus target="#25 #30v #31r"/>. The upper edge might have been slightly damaged by fire.
-                                    All edges have been trimmed.</condition>
+                                <locus target="#25 #30v #31r"/>. The upper edge might have been slightly damaged by fire.</condition>
```

Comment:
It is hard to tell from the pictures with any degree of certainty if this is the case, but fire damage usually causes the parchment to shrink and wrinkle ... here I would see "only" (heavy) insect droppings

Reply:
Thank you! I've added it, but of course it would be good if you revised the existing condition descriptions in your own workflow, clearly they will become much better!

Reply:
Thank you Dorothea!I will do it!